create-job-ui/src/components/Textfeild.js

```diff
@@ -7,7 +7,7 @@ export const Textfeild = ({placeholderText,inputValue,setInputValue}   ) => {
           {/* <input type="text" value={inputValue} onChange={e => setInputValue(e.target.value)} /> */}
           <label>
             
-        <input type="text" placeholder={placeholderText}  value={inputValue} onChange={e => setInputValue(e.target.value)} />
+        <input type="text" placeholder={placeholderText}  value={inputValue} onChange={e => setInputValue(e.target.value)} required/>
```

Comment:
@asharMehmood2211004KHICND I assume not all uses of this input will be `required`. We can provide this as a prop e.g `required={htmlRequired`